conda_build/utils.py

```diff
@@ -950,11 +950,10 @@ def get_build_folders(croot):
 
 
 def prepend_bin_path(env, prefix, prepend_prefix=False):
-    # bin_dirname takes care of bin on *nix, Scripts on win
-    env['PATH'] = join(prefix, bin_dirname) + os.pathsep + env['PATH']
+    env['PATH'] = join(prefix, "bin") + os.pathsep + env['PATH']
```

Comment:
`conda activate` always adds `"bin"` to `PATH`. We have to have it here too. The addition of `"Scripts"` is taken care of a few lines below.